src/Silex/Provider/Routing/RedirectableUrlMatcher.php

```diff
@@ -24,7 +24,7 @@ class RedirectableUrlMatcher extends BaseRedirectableUrlMatcher
     /**
      * {@inheritdoc}
      */
-    public function redirect($path, $route, $scheme = null)
+    public function redirect(string $path, string $route, string $scheme = null): array
```

Comment:
It won't work for an older version of `symfony/routing`, please fix it the same way as for other cases.